backend/entrypoint.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+ln --symbolic \
```

Comment:
There isn't the `symbolic` flag in the `ln` from busybox
```suggestion
ln -s \
```